tests/libnet/hotplug.go

```diff
@@ -119,11 +119,16 @@ func cleanMACAddressesFromStatus(status []v1.VirtualMachineInstanceNetworkInterf
 func interfaceStatusFromInterfaces(queueCount int32, ifaces []v1.Interface) []v1.VirtualMachineInstanceNetworkInterface {
 	const (
 		initialIfacesInVMI = 1
-
-		linkStateUp = "up"
 	)
 	var ifaceStatuses []v1.VirtualMachineInstanceNetworkInterface
 
+	normalizeState := func(state v1.InterfaceState) string {
```

Comment:
Please consider converting this function into a plain function as it is not used as a closure.

Reply:
Actually, it is still here.